app/javascript/utils/formUtils.js

```diff
@@ -49,6 +49,11 @@ export const maxLengthMap = {
   income: 16
 }
 
+export const reorderEmptyOptions = (options) => {
+  options.some((option, i) => isEmpty(option.value) && options.unshift(option) && options.splice(i + 1, 1))
```

Comment:
Loop through and find an empty value, unshift it to the front, and splice the old index out